src/pages/create/tabs/ReviewDeployTab/DeployProjectButton.tsx

```diff
@@ -110,13 +100,47 @@ export default function DeployProjectButton() {
       return
     }
 
-    const groupedSplits = [payoutGroupedSplits, reservedTokensGroupedSplits]
-    if (nftRewardsCIDs) {
-      // TODO: Set loading states true of nftTxGas and launchTxGas.
-      // Call gas reader for gas of nft tx and launch tx within multicall.
-      // Set value and loading states of nftTxGas and launchTxGas.
-      // setGasReaderModalVisible(true)
+    const txOpts = {
```

Comment:
@johnnyd-eth might we just revert this file to what it was, sans-nft?

Just so we (I) don't miss anything/ensure nothing changes.

Reply:
Yep will do

Reply:
thank youuuuuu